Makefile

```diff
@@ -34,32 +33,6 @@ service: gx-deps
 ctl: gx-deps
```

Comment:
Don't use removed targets `docker_gx-deps`, `gx-deps` (here and at all other places)